src/utils/secretManagerService.test.ts

```diff
@@ -134,6 +135,18 @@ test('should return null if MPS certs does not exists', async () => {
   expect(secretAtPathSpy).toHaveBeenCalledWith('MPSCerts')
 })
 
+test('should remove device from secret store', async () => {
+  gotDeleteSpy = jest.spyOn(secretManagerService.gotClient, 'delete').mockImplementation(() => {
+    return {
+      json: jest.fn(() => {
+        return null
+      })
+    } as any
+  })
+  await secretManagerService.deleteSecretAtPath('test')
+  expect(gotDeleteSpy).toHaveBeenCalled()
```

Comment:
only thing i would change here is to ensure the call was made with what is expected. So:

``` suggestion
expect(gotDeleteSpy).toHaveBeenCalledWith('test',{ prefixUrl: 'theexpectedurl'})
```